client-src/elements/chromedash-all-features-page.ts

```diff
@@ -2,7 +2,20 @@ import {LitElement, css, html, nothing} from 'lit';
 import {showToastMessage} from './utils.js';
 import './chromedash-feature-table';
 import {SHARED_STYLES} from '../css/shared-css.js';
+import {customElement, property, queryAll, state} from 'lit/decorators.js';
+import {User} from './chromedash-activity-log.js';
 
+interface RawQuery {
+  q?: string;
+  columns?: string;
+  showEnterprise?: boolean;
+  sort?: string;
+  start?: string;
+  num?: string;
+  [key: string]: any;
```

Comment:
Define this in `utils.js` next to https://github.com/GoogleChrome/chromium-dashboard/blob/9bff5bc3b05d091b5f1e7270175091222ba72aa9/client-src/elements/utils.js#L357.

All fields should actually be strings, since that's what comes out of URLSearchParams.getAll().